Source/Core/Core/HW/DSPHLE/UCodes/AX.cpp

```diff
@@ -741,13 +753,15 @@ void AXUCode::HandleMail(u32 mail)
     break;
 
   case MailState::WaitingForCmdListAddress:
+  {
     CopyCmdList(mail, m_cmdlist_size);
-    HandleCommandList();
-    m_cmdlist_size = 0;
-    SignalWorkEnd();
+    auto& core_timing = m_dsphle->GetSystem().GetCoreTiming();
+    // Cycle count determined by ear for Donkey Kong Barrel Blast.
+    core_timing.ScheduleEvent(10'000, m_event_type_handle_command_list, (u64)this);
```

Comment:
Passing a pointer as userdata breaks savestates.

Reply:
Good catch, should be fixed now.